include/SDL3/SDL_stdinc.h

```diff
@@ -1291,6 +1291,21 @@ extern SDL_DECLSPEC int SDLCALL SDL_strcasecmp(const char *str1, const char *str
  */
 extern SDL_DECLSPEC int SDLCALL SDL_strncasecmp(const char *str1, const char *str2, size_t maxlen);
 
+/**
+ * Scans the null-terminated string pointed to by str for any character
+ * from the null-terminated string pointed to by accept,
+ * and returns a pointer to that character.
+ *
+ * \param str null-terminated string in which characters are searched.
+ * \param accept null-terminated string containing characters to search for.
```

Comment:
This seems like an odd name for this parameter. Let's go with str1 and str2, which are used in all the online references for this function.

Reply:
Also, what format are these characters? ASCII? UTF-8? It looks like the C library function uses ASCII, so if we're supporting UTF-8, then we should probably roll our own implementation.

Reply:
The libc documents them as null-terminated byte strings. I'm going to change the documentation to that.

glibc uses `s` and `accept`.
cppreference.com uses `dest` and `break`.

`str1` and `str2` is not a good suggestion: there's a difference between them.